ex35.py

```diff
@@ -63,49 +65,49 @@ def cthulu_room():
         cthulu_room()
 
 
-tries = 0 # how to get this into the function and used in the nested functions?
-          # maybe not possible? blech
 def cthulus_mind():
+    tries = 0
+
+    while tries < 11:
+        if tries == 0:
+            print "Your vision begins to blur and goes dark."
+            print "You hear in the distance a familiar bellow:"
+            print "Mortal... completing this task will prove your sanity in the face of a god..."
+            print "ENTERRRRRRRRRRRR"
+            cthulu_mind_test()
+            tries += 1
+        elif 0 < tries < 10:
+            print "You may try again. Beware, these are not unlimited..."
+            print 10 - tries
+            cthulu_mind_test()
+            tries += 1
+        else:
+            print "You go insane and begin to flee!"
+            start()
+    return
 
-    if tries == 0:   
-        print "Your vision begins to blur and goes dark."
-        print "You hear in the distance a familiar bellow:"
-        print "Mortal... completing this task will prove your sanity in the face of a god..."
-        print "ENTERRRRRRRRRRRR"
-        cthulu_mind_test()
-    elif 1 <= tries <= 10:
-        print "You may try again. Beware, these are not unlimited..."
-        print 10 - tries
-        cthulu_mind_test()
-    else:
-        print "You go insane and begin to flee!"
-        start()
 
-def cthulu_mind_test(): # this really should be its own function - complete
+def cthulu_mind_test():
     print "Suddenly arcane rune symbols enter your field of vision"
     print "You recognize them! Arithmitic..."
     print "You then hear a deep echo:"
     print "What is EIGHT to the power of TWO?!!"
-    choice = int(raw_input("> "))

```

Comment:
If the input read by the raw_input() method here could not be cast to an int, it would throw a ValueError exception, which would not be caught (because it's outside the scope of a try-block), and your program would crash.